app/models/host/managed.rb

```diff
@@ -653,6 +653,19 @@ def clone
     host
   end
 
+  def api_clone
+    host = self.selective_clone
+    if self.interfaces.length == 1
```

Comment:
It clones the interfaces if there is only 1. If there are multiple, then nothing gets cloned, so the whole call fails on duplicate DHCP entry for me.